File: ADVANCED BUSINESS DEVELOPMENT WITH .NET/src/Domain/Entities/Clinica.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace UserApi.Domain
{
    public class Clinica
    {
        [BsonId]
        public ObjectId Id { get; set; }  // ID do banco
        public string? IdClinica { get; set; } = null!; 
        public string Nome { get; set; } = null!;
        public string Endereco { get; set; } = null!;
        public string Telefone { get; set; } = null!;
        public string Avaliacao { get; set; } = null!;
        public float PrecoMedio { get; set; }
        public string Email { get; set; } = null!;
}
}

File: JAVA ADVANCED/sprint-2/challenge/src/main/java/br/com/fiap/challenge/domains/Clinica.java
package br.com.fiap.challenge.domains;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Entity
@Data
@Builder
@Table(name = "clinica")
public class Clinica {

    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    @Column(name = "id_clinica", nullable = false)
    private String  idClinica;

    @Column(name = "nome", length = 100, nullable = false)
    private String nome;

    @Column(name = "endereco", length = 250, nullable = false)
    private String endereco;

    @Column(name = "telefone", length = 15, nullable = false)
    private String telefone;

    @Column(name = "avaliacao", nullable = false)
    private Float avaliacao;

    @Column(name = "preco_medio", nullable = false)
    private Double precoMedio;

}
